### DISCUSSION TITLE
Superb 2022 FL - DCC slider coding issue

### DISCUSSION THREAD

**Discussion by Szo-gun** [jilleb/MQB-FPA#96] [Q&A] [answered]
Hello,

I want to improve my DCC settings - from 3 position to slider in Superb 2020.
But whatever I do I cannot change it to slider.

What have I done:
1. Flashed DCC.  From 5Q0907376B to 5Q0907376C.
2. Updated DCC parameters to Arteon parameters.
(I'v used original parameters from VCP library - maybe this will be cause of the issue. Maybe not all Arteons have slider?)
3. Changed parameters in FPA Editor:
![image](https://github.com/jilleb/MQB-FPA/assets/66619283/042fc603-7b51-468f-bb28-00ee63115854)
4. Uploaded it to the gateway.
5. Reset unit.

But still looks like this:
![image](https://github.com/jilleb/MQB-FPA/assets/66619283/217e6d02-adc8-40f2-ad51-3afa4050f403)

Do I need to change something more in FPA Editor? Or should I try with other dataset for DCC module?

*It looks like parameters for gateway are working - because I can change other settings - for example dynamic lights, sport diff...

**Comment by jilleb** [comment, marked as answer]
Make sure to add all profiles to the individual profile as well. Adding to the request values alone kant enough. I put some helpful tips to achieve dcc slider in the help texts on mqbtools.nl
